content-services/latest/install/ansible.md

```diff
@@ -61,110 +63,104 @@ all:
-  repository clustering.
+* **HA** (also known as `ha` - high availability). It's very similar to the `ssh` one but also provides a skeleton for repository clustering.
+
+A complete documentation about inventory file is available at [Ansible](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html#intro-inventory).
 
 ## Prerequisites
-If you're using the Enterprise edition of Content Services, then you need credentials to access the necessary artifacts from 
-[Nexus](https://artifacts.alfresco.com/nexus/){:target="_blank"}. Customers can request these through 
-[Hyland Community](https://community.hyland.com/){:target="_blank"}.
+
+If you're using the Enterprise edition of Content Services, then you need credentials to access the necessary artifacts from [Nexus](https://artifacts.alfresco.com/nexus/){:target="_blank"}. Customers can request these through [Hyland Community](https://community.hyland.com/){:target="_blank"}.
 
 ### Control Node
-In addition to the requirements mentioned earlier for control nodes in general, the playbook requires Ansible 2.12+, 
-which in turn requires python 3.8+.
+
+In addition to the requirements mentioned earlier for control nodes in general, the playbook requires Ansible 2.12+, which in turn requires python 3.8+.
 
 ### Target O/S
-While Content Services supports a wide range of OS, the playbook is only supported for a subset of them. The table below 
-gives detailed information on the status of supported OS (Additional target environments will be added in future releases):
-
-| OS Flavor / version | 7.2 Enterprise| 7.1 Enterprise | 7.0 Enterprise | 6.2.2 Enterprise | Community |
-|-|-|-|-|-|-|
-| Amazon Linux (v2) | - | - | - | - | - |
-| Amazon Linux (v1) | - | - | - | - | - |
-| RHEL 8.4 | [x] |[x] |[x] | - | [x] |
-| RHEL 8.2 | [x] |[x] |[x] | - | [x] |
-| RHEL 7.7 | - |[x] |[x] | - | [x] |
-| RHEL 7.6 | - |[x] | [x] | [x] | [x] |
-| CentOS 8 x64 | [x] |[x] |[x] | - | [x] |
-| CentOS 7 x64 | [x] |[x] |[x] | [x] | [x] |
-| Ubuntu 20.04 | [x] |[x] |[x] | - | [x] |
-| Ubuntu 18.04 | - |[x] |[x] | [x] | [x] |
-| SUSE 15.0 | - | - | - | - | - |
-| SUSE 12.0 SP1 x64 | - | - | - | - | - |
+
+While Content Services supports a wide range of OS, the playbook is only supported for a subset of them. The table below gives detailed information on the status of supported OS (Additional target environments will be added in future releases):
+
+| OS Flavor / version |  7.3 Enterprise | 7.2 Enterprise | 7.1 Enterprise | 7.0 Enterprise | 6.2.2 Enterprise | Community |
+|-|-|-|-|-|-|-|
+| Amazon Linux (v2) | - | - | - | - | - | - |
+| Amazon Linux (v1) | - | - | - | - | - | - |
+| RHEL 8.6 | [x] | - | - | - | - | [x] |
+| RHEL 8.5 | [x] | [x] | - | - | - | [x] |
+| RHEL 8.4 | [x] | [x] | [x] |[x] | - | [x] |
+| RHEL 8.2 | [x] | [x] | [x] | - | [x] |
+| RHEL 7.7 | - | - | [x] | [x] | - | [x] |
+| RHEL 7.6 | - | - | [x] | [x] | [x] | [x] |
+| CentOS 8 x64 | [x] | [x] |[x] |[x] | - | [x] |
```

Comment:
That's a mistake on our side. Centos 8 is not supported anymore for 7.3. You can actually remove the line completely as we have removed support for it in the latest playbook version. If a customer wants to deploy on centos 8, he would need to use an older version of the playbook

Reply:
I removed the CentOS 8 line.